content/docs/06-integrations/00-spectro-k8s-dashboard.md

```diff
@@ -0,0 +1,144 @@
+<br />
+
+#### User Selections
+
+These settings are also included in the manifest as `access` and `identityProvider` parameters.
+
+| Parameter | Description | Default |
+|-----------|-------------|---------|
+| Access | Controls cluster access. Private access means that nodes and pods are isolated from the internet. | Private |
+| Identity Provider | You can use Palette as the IDP or a third-party application as the IDP. | Palette |
+
+#### Internal Manifest Parameters
+
+| Parameter | Description | Default |
+|-----------|-------------|---------|
+| namespace | The Kubernetes namespace to install the dashboard. | `kubernetes-dashboard` |
+| ClusterRole | The ClusterRole to assign to the Kubernetes Dashboard. | `read-only` |
+| certDuration | Self-signed certificate duration in hours. | 8760h (365d) |
+| certRenewal | Self-signed certificate renewal in hours | 720h (30d) |
+| enableInsecureLogin | RBAC ensures secure login. | `false` |
+| serviceType | The service type for the dashboard. Supported values are ClusterIP, LoadBalancer, and NodePort. | `ClusterIP` |
+| skipLogin | Enables or disables the skip login option on the dashboard. | `false` |
+| enableInsecureLogin | Enables non-Secure Sockets Layer (SSL) login. Dashboard login is always restricted to HTTP(S) + localhost or HTTPS and external domain. | `false` |
+| ingress:enabled | Ingress configuration to access the ClusterIP, loadBalancer, or nodePort. | `false` |
+
+## Usage
+
+To use the Spectro Kubernetes Dashboard pack, you have to add it to your cluster profile. The Spectro Kubernetes Dashboard pack has default settings for **Access** and **Identity Provider**. These settings automatically install the latest version of the Spectro Proxy pack when the cluster is created.
+
+Changing these default settings may require some additional configuration.
+<br /> 
+
+### Access
+
+The default setting is **Private**. If you change the setting to **Public** and your cluster is in a public cloud, there is no additional configuration.  
+
+However, if you change **Access** to **Public** and your cluster is in a private cloud, you have to manually add and configure the Spectro Proxy pack to your cluster profile. For more information, refer to the [Spectro Proxy](https://docs.spectrocloud.com/integrations/frp) guide.
+<br /> 
+
+### Identity Provider 
+
+The default setting is **Palette**. If you change this setting to **Inherit from Tenant**, you must configure OpenID Connect (OIDC) authentication. You can configure OIDC two ways:
+
+- In Tenant Admin scope, navigate to **Tenant Settings > SSO**, choose **OIDC**, and provide your third-party Identity Provider details.  
+
+- In **Tenant Settings > SSO**, if you choose **None** or **SAML**, you must configure OIDC manually in the Kubernetes pack. 
+
+### Enable OIDC Authentication
+
+To enable OIDC authentication manually, copy the configuration shown in the example to the Kubernetes Pack and provide your third-party Identity Provider details.
```

Comment:
These parameters are already in the Kubernetes pack YAML definition. The user just needs to uncomment them, so there is no need to copy them.

Reply:
@karl-cardenas-coding I don't see the "OIDC related config" in the config file. 

@ravi-k8 @shantanubansal Can you please confirm? If a user changes the default value to 'Inherit from tenant', do they have to add this "OIDC related config" block?

Reply:
@ritawatson I think the label should be "Inherit from Org"
If user selects "Inherit from Org" then backend would take care of adding client config block at run time. User is not required to add additional configuration.

Reply:
@ravi-k8 Has the UI label been changed? I'm going by the demo that was given. The UI at the time said 'Inherit from Tenant'.

Reply:
@ritawatson Yes, label is updated to `Inherit from Organization`